src/pages/projectSettings/tabs/EventLogTab/EventLogTab.tsx

```diff
@@ -38,11 +55,112 @@ export const EventLogTab: React.FC = () => {
     },
   });
 
+  const eventData: LogEntry[] = isRepo
```

Comment:
When I load a valid project that I suppose lacks event data, I see this error:
<img width="1370" alt="image" src="https://user-images.githubusercontent.com/9298431/165767494-5ed56f4a-9c53-4712-ad5a-20ce47a7c47a.png">

Do you think it could be cleaned up?

Reply:
yes, but let's do it in a separate ticket.

Reply:
wait no, I misunderstood. I'll do that now.

Reply:
This should be solved on the backend, the resolver shouldn't be returning an error in this case. I'll create a ticket.